src/style.css

```diff
@@ -1 +1,110 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap");
+
+html,
+body,
+div,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+a {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+html,
+body {
+  height: 100%;
+  width: 100%;
+  font-style: 100%;
+  line-height: 1;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+button {
+  cursor: pointer;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+ul li {
+  list-style-type: none;
+}
+
+img {
+  vertical-align: top;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-style: inherit;
+  font-weight: 400;
+}
+
+.logo__img {
+  margin-left: 50px;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.menu {
+  display: flex;
+  justify-content: space-between;
+}
+
+.menu__item {
+  margin: 23px 0 23px 20px;
```

Comment:
Please reset margin for first element because of this, most of the tests you have failed
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/752a04d2-5086-4be3-a8e9-69866d0bee2e)

Also the clickable area of your menu item is limited to text, because u use margin instead of padding, try to make it the full height of the header
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/2a4268bd-1dee-4455-93d0-a94eca26a4e2)